test/graphql/GraphQLTemplatesTest.js

```diff
@@ -0,0 +1,154 @@
+/*******************************************************************************
+ *
+ *    Copyright 2018 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+
+'use strict';
+
+const cutIndex = __dirname.indexOf('test');
+const path = __dirname.substring(0, cutIndex) + 'src/graphql';
+const handlebars = require('handlebars');
+handlebars.registerHelper('ifCond', function (v1, operator, v2, options) {
```

Comment:
The test shouldn't redefine this as it's implemented in `GraphQlRequestBuilder.js` so you should not duplicate that code in the test